components/captcha-challenge.tsx

```diff
@@ -77,6 +78,11 @@ const CaptchaChallenge: React.FC<{ phoneNumber: string }> = ({ phoneNumber }) =>
 
   useEffect(() => {
     const initCaptcha = async () => {
+      try {
+        await clearCookies()
+      } catch (e) {
+        console.debug("[Init captcha err]:", "no cookies to clear")
+      }
```

Comment:
why not include it in the other trycatch block?

Reply:
i think it returns a 500 if no cookies and i still want it to proceed to the captcha code. not sure if the browser fetch throws on a 500 or not

Reply:
actually just looked it up, it does not throw on 500s (maybe its axios that does that).  i will move it into the main try